xlive/Blam/Engine/game/game_engine.h

```diff
@@ -146,14 +146,25 @@ class c_game_engine
 struct s_game_engine_global_player_info
 {
 	bool valid;
-	byte pad_1[3];
+	uint8 pad_1[3];
 	real_point3d point;
 	short field_10;
 	short field_12;
-	byte field_14[4];
+	uint8 field_14[4];
 };
 ASSERT_STRUCT_SIZE(s_game_engine_global_player_info, 24);
 
+struct s_simulation_player_neddebug_data
+{
+	int32 field_0;
+	int32 field_4;
+	int16 rtt_msec;
+	int16 pck_rate;
+	int16 throughput;
+	int16 pck_loss;
+};
+ASSERT_STRUCT_SIZE(s_simulation_player_neddebug_data, 16);
```

Comment:
Same size as HOs `s_simulation_player_netdebug_data`, probably the same data.

Reply:
not sure about the first 2 fields, will keep them this way for now